draft-esdih-ccamp-layer0-types-ext-00.txt

```diff
@@ -0,0 +1,1344 @@
+   See RFC 3552 [RFC3552] for a guide.
+
+8.  References
+
+8.1.  Normative References
+
+   [I-D.ietf-ccamp-dwdm-if-param-yang]
+              Galimberti, G., Kunze, R., Burk, A., Hiremagalur, D., and
+              G. Grammel, "A YANG model to manage the optical interface
+              parameters for an external transponder in a WDM network",
+              Work in Progress, Internet-Draft, draft-ietf-ccamp-dwdm-
+
+
+
+Beller, et al.           Expires 26 August 2021                [Page 22]
+
+Internet-Draft         Layer 0 Types - Revision 2          February 2021
+
+
+              if-param-yang-05, 18 November 2020,
+              <https://tools.ietf.org/html/draft-ietf-ccamp-dwdm-if-
+              param-yang-05>.
+
+   [I-D.ietf-ccamp-layer0-types]
+              Zheng, H., Lee, Y., Guo, A., Lopez, V., and D. King, "A
+              YANG Data Model for Layer 0 Types", Work in Progress,
+              Internet-Draft, draft-ietf-ccamp-layer0-types-09, 29
+              December 2020, <https://tools.ietf.org/html/draft-ietf-
+              ccamp-layer0-types-09>.
+
+   [I-D.ietf-ccamp-optical-impairment-topology-yang]
+              Lee, Y., Auge, L., Lopez, V., Galimberti, G., and D.
+              Beller, "A Yang Data Model for Optical Impairment-aware
+              Topology", Work in Progress, Internet-Draft, draft-ietf-
+              ccamp-optical-impairment-topology-yang-05, 2 November
+              2020, <https://tools.ietf.org/html/draft-ietf-ccamp-
+              optical-impairment-topology-yang-05>.
+
+   [I-D.ietf-ccamp-wson-yang]
+              Zheng, H., Lee, Y., Guo, A., Lopez, V., and D. King, "A
+              YANG Data Model for WSON (Wavelength Switched Optical
+              Networks)", Work in Progress, Internet-Draft, draft-ietf-
+              ccamp-wson-yang-28, 29 December 2020,
+              <https://tools.ietf.org/html/draft-ietf-ccamp-wson-yang-
+              28>.
+
+   [RFC2119]  Bradner, S., "Key words for use in RFCs to Indicate
+              Requirement Levels", BCP 14, RFC 2119,
+              DOI 10.17487/RFC2119, March 1997,
+              <https://www.rfc-editor.org/info/rfc2119>.
+
+   [RFC6241]  Enns, R., Ed., Bjorklund, M., Ed., Schoenwaelder, J., Ed.,
+              and A. Bierman, Ed., "Network Configuration Protocol
+              (NETCONF)", RFC 6241, DOI 10.17487/RFC6241, June 2011,
+              <https://www.rfc-editor.org/info/rfc6241>.
+
+   [RFC7950]  Bjorklund, M., Ed., "The YANG 1.1 Data Modeling Language",
+              RFC 7950, DOI 10.17487/RFC7950, August 2016,
+              <https://www.rfc-editor.org/info/rfc7950>.
+
+   [RFC8795]  Liu, X., Bryskin, I., Beeram, V., Saad, T., Shah, H., and
+              O. Gonzalez de Dios, "YANG Data Model for Traffic
+              Engineering (TE) Topologies", RFC 8795,
+              DOI 10.17487/RFC8795, August 2020,
+              <https://www.rfc-editor.org/info/rfc8795>.
```

Comment:
I think we can remove this reference: RFC8795 is not mentioned in the draft text